openrtb_ext/user.go

```diff
@@ -10,6 +10,8 @@ type ExtUser struct {
 	// https://iabtechlab.com/wp-content/uploads/2018/02/OpenRTB_Advisory_GDPR_2018-02.pdf
 	Consent string `json:"consent,omitempty"`
 
+	ConsentedProvidersSettings *ExtUserCPSettings `json:"ConsentedProvidersSettings,omitempty"`
```

Comment:
Is it a reason json mapping value starts from uppercase? Feels like it should start from lowercase.

Reply:
It does look odd but I used the exact format described in [issue #1817](https://github.com/prebid/prebid-server/issues/1817) and that's also how it was implemented in Prebid Server Java. Below an excerpt from a PBS-Java JSON test file:
```
 70   "user": {
 71     "buyeruid": "J5VLCWQP-26-CWFT",
 72     "ext": {
 73       "ConsentedProvidersSettings": {
 74         "consented_providers": "someConsent"
 75       }
 76     }
 77   },
./src/test/resources/org/prebid/server/it/amp/test-rubicon-bid-request.json
```

Reply:
json mapping is not case sensitive, it should work with `ConsentedProvidersSettings` and `consentedproviderssettings`. We can discuss it with the team.